src/Recommendations/Rule/BidirectionalRelationship.php

```diff
@@ -66,8 +66,13 @@ protected function getRelatedArticles(string $id): Sequence
     public function resolveRelations(RuleModel $input): array
     {
         $this->logger->debug('Looking for '.$this->type.' articles related to Article<'.$input->getId().'>');
-        $related = $this->getRelatedArticles($input->getId());
-        if ($related->count() === 0) {
+        try {
+            $related = $this->getRelatedArticles($input->getId());
+
+            if ($related->count() === 0) {
+                return [];
+            }
+        } catch (\Throwable $e) {
```

Comment:
the exception is ignored, ok to fall back to an empty response but we should log it at least